amethyst_audio/src/components/audio_emitter.rs

```diff
@@ -53,3 +54,79 @@ impl AudioEmitter {
 impl Component for AudioEmitter {
     type Storage = BTreeStorage<Self>;
 }
+
+#[cfg(test)]
+mod tests {
+    use crate::{AudioEmitter, Source};
+    use amethyst_utils::app_root_dir::application_root_dir;
+    use std::{fs::File, io::Read, vec::Vec};
```

Comment:
by the way, could you add empty newlines between import groups like this:

```rust
use std::{..};
// newline
use amethyst_utils::{..};
use another::{..};
// newline
use crate::{..};
```

we adopted the convention to quickly see what's from std / other crates / current crate.

oh ya, are you used to `use`ing `std::{boxed::Box, vec::Vec}`? (guessing that's standard practice for a `#[no_std]` crate?)
normally we leave them out as well

Reply:
I saw the empty new lines in the other src files. Can change for tests as well.

I'm not used to anything I'm new to the language. Happy to help keep conventions consistent.